source/reference/modules/engine/index.rst

```diff
@@ -1330,6 +1330,18 @@ Crafter Engine sports a built-in cache engine with an LRU (least recently used)
 
 .. note:: When running in Preview Mode (inside Studio for preview purposes), Crafter Engine's cache is disabled to help authors see their changes immediately.
 
+Engine has multiple caches:
+
+.. version_tag::
+    :label: Since
+    :version: 4.3.1
+
+- **user** cache for objects cached by Groovy code.
+  This cache is cleared manually by calling the :base_url:`cacheClear <_static/api/engine.html#tag/cache/operation/cacheClear>` API.
+  Active cache is only used on the user cache.
```

Comment:
Active cache is only used in the application cache when the cache is enabled, if not enabled Engine behaves like in previous versions, with active cache running in the system cache.